### ISSUE TITLE
[Feature Request]  option --no-statusbar

### ISSUE THREAD

**Issue by nkh** [walles/moar#93]
an option to NOT display any statusbar and a shortcut to toggle between not showing it and whatever --statusbar is set to

**Comment by walles** [comment]
In what situation do you want to hide the statusbar?

When / how is the current statusbar in the way / unhelpful?

**Comment by nkh** [comment]
All the time!

the statusbar is just a distraction that costs me a line of display

there may be times that I am interested to know that I am at 37% of the file but that's 1% of the time and then I'd like to ask for it

I dropped in moar as a replacement for less  in an application I'm writing, the status bar (compared to less ":") was mehhh but that line on white background on my black terminal got moar removed very quickly.

I continued playing with it to see if it is a worthwhile replacement for  "less". I'm not convinced yet but I think new applications need to be given a serious try otherwise we'd never change to the better. Not that "moar" is not capable just that it's difficult to find a good reason; I also profited of the occasion to give some feedback, because I really want to  use new applications ;). 

The same goes for the unsightly ">" for continuation, a trait shared by "less" unfortunately.

moar:
![screenshot_2022-07-20_09-38-09](https://user-images.githubusercontent.com/45409/179925360-5c3c21cb-84e1-4002-a281-5344a21ec53f.png)

less:
![screenshot_2022-07-20_09-40-06](https://user-images.githubusercontent.com/45409/179925429-9e5c520e-a50d-4b48-99c3-c1f216925ec4.png)
